src/main/java/core/basesyntax/service/FileEntryReaderImpl.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax.service;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class FileEntryReaderImpl implements FileEntryReader {
+    private static final String ERROR_MESSAGE = "Can't read the file in this path";
+    private static final int SKIP_TITLE = 1;
+
+    @Override
+    public List<String> readFile(String path) {
+        try {
+            return Files.readAllLines(Path.of(path))
+                    .stream()
+                    .skip(SKIP_TITLE)
```

Comment:
we are reading the file and we don't know that this file is csv - why should we skip the first line?

let's skip this line in parsing service

Reply:
Done